automation/taskcluster/lib/tasks.py

```diff
@@ -130,25 +130,25 @@ def craft_test_task(self, variant):
     def _craft_branch_routes(self, variant):
         routes = []
 
-        if self.repo_url == _OFFICIAL_REPO_URL and self.branch == 'master':
+        if self.repo_url == _OFFICIAL_REPO_URL and self.git_ref == 'refs/heads/master':
             architecture, build_type, product = \
                 _get_architecture_and_build_type_and_product_from_variant(variant)
             product = convert_camel_case_into_kebab_case(product)
             postfix = convert_camel_case_into_kebab_case('{}-{}'.format(architecture, build_type))
 
             routes = [
                 'index.project.mobile.fenix.branch.{}.revision.{}.{}.{}'.format(
-                    self.branch, self.commit, product, postfix
+                    self.git_ref, self.commit, product, postfix
```

Comment:
Oof, the index naming is off: `.branch` instead of `.ref`/`.git_ref`.

Also, does it really look like `index.project.mobile.fenix.branch.refs/heads/master.latest.mobile.blah`? Seems weird to have slashes in there :thinking:

Reply:
Oooooooooh, nice catch! On it.

Reply:
Fixed it by using `short_head_branch`, which is processed in `.taskcluster.yml`. I didn't want to replicate the logic. That's why both `MOBILE_HEAD_BRANCH` and `SHORT_HEAD_BRANCH` are passed down

Reply:
Beauty! That's awesome :100: